bin/IDFilter.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+# Written by Jonas Scheid under the MIT license
+
```

Comment:
Code seems fine, but is there not some functionality in openms that is doing this?

Reply:
This has been implemented in the IDFilter of [OpenMS dev](https://github.com/OpenMS/OpenMS/pull/7021), however will only be released with 3.1.0, which could take some time. The pyopenms IDfilter already has this implementation. So once OpenMS 3.1.0 is released, this python script will be replaced by IDfilter

Reply:
Then it might be good to include this as an issue: altering the Pyopenms IDfilter when OpenMS 3.1.0 is released, or something like that

Reply:
https://github.com/nf-core/mhcquant/issues/283